.github/workflows/ci-core.yml

```diff
@@ -67,6 +67,8 @@ jobs:
             ${{ runner.os }}-build-
             ${{ runner.os }}-
       - run: yarn install --frozen-lockfile
+      - name: Check Config Docs
+        run: diff ./docs/CONFIG.md <(go run ./core/services/chainlink/cmd/doc/doc.go) || echo "./docs/CONFIG.md is out of date. Run 'make config' to regenerate."
```

Comment:
A Go test would be simpler and trigger sooner. I think it would be easy to rig, but would need a messy relative path prefix (`../../../..`) in the current state :thinking:

Reply:
Fixed